src/app/pages/home-page/offer-list/offer-list.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
in develop we have offer-list-page component. you dont need to commit this file. delete it locally and make push

Reply:
Hmmm, it's don't work without these imports in my local branch...